### PR TITLE
Setting up urbansim recipe

### PR DESCRIPTION
**Pull request by smmaurer**
Hi! I'm setting up a recipe for the [UrbanSim](https://github.com/UDST/urbansim) library, part of the Urban Data Science Toolkit. I'm the lead maintainer.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/urbansim/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
```
Comment (synapticarbors) [inline recipes/urbansim/meta.yaml:23]: The upstream `setup.py` specifies a number of version restrictions on the dependencies:

https://github.com/UDST/urbansim/blob/dev/setup.py#L32-L42

Unless there is a good reason to ignore it, we should include them here.

File: recipes/urbansim/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    - patsy
+    - prettytable
+    - pyyaml
+    - scipy
+    - statsmodels
+    - toolz
+
+test:
+  imports:
+    - urbansim
```
Comment (synapticarbors) [inline recipes/urbansim/meta.yaml:37]: Also import submodules:

```
   - urbansim
   - urbansim.developer
   - urbansim.maps
   - urbansim.models
   - urbansim.urbanchoice
   - urbansim.utils
```

File: recipes/urbansim/meta.yaml (hunk 1 above)
Comment (smmaurer) [reply recipes/urbansim/meta.yaml:23]: Added!

I didn't include the detail where `statsmodels` depends on the Python version -- this is a pip-specific issue that I haven't run into with conda. (Statsmodels v0.11 doesn't run in Python <3.6, but pip tries to install it anyway.)

File: recipes/urbansim/meta.yaml (hunk 2 above)
Comment (smmaurer) [reply recipes/urbansim/meta.yaml:37]: Added!

**Comment by smmaurer** [comment]
Hi @synapticarbors, thank you for the review!

**Comment by smmaurer** [comment]
Hi @synapticarbors, it looks like something failed after this was merged, but I'm not really sure how this is supposed to work. Do you have any advice?

https://github.com/conda-forge/staged-recipes/runs/656990976

**Comment by synapticarbors** [comment]
Ok, thanks for letting me know. I was told by another developer to wait for an hour or so and they'll try to restart the Travis server and see if that fixes it. If you don't see it fixed in the next bit, ping me again.

**Pull request merged**